src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Avatar
+description: >
+  Avatars display typography, icons, or images in different sizes. They help users quickly recognize
+  an entity.
+package: '@zendeskgarden/react-avatars'
+propSheets:
+  - avatars/src/elements/Avatar.tsx
+---
+
+<Usage>
+  <div>
+    <MD isBold>When to use</MD>
+    <ul type="disc">
```

Comment:
This should be consistent with our other `<ul>` usages in markdown. If `type="disc"` should be the default we can default that attribute in `typography/index.ts`